src/screens/LocationPage/utils.ts

```diff
@@ -1,27 +1,18 @@
-import { Projections } from 'common/models/Projections';
-import { findLocationForFips, findStateByFips } from 'common/locations';
 import { LOCATION_SUMMARY_LEVELS } from 'common/metrics/location_summary';
 import { formatMetatagDate } from 'common/utils';
+import { Projections } from 'common/models/Projections';
+import { Region } from 'common/regions';
 
-function getLocationName(fipsCode: string) {
-  const location = findLocationForFips(fipsCode);
-  const { state: stateName } = findStateByFips(location.state_fips_code);
-  const { county: countyName, state_code: stateCode } = location;
-  const locationName = location.county
-    ? `${countyName}, ${stateName} (${stateCode})`
-    : `${stateName} (${stateCode})`;
-  return locationName;
-}
-
-export function getPageTitle(projections: Projections): string {
-  const locationName = getLocationName(projections.fips);
-  return `${locationName} - COVID Risk Map & Key Metrics`;
+export function getPageTitle(region: Region): string {
+  return `${region.shortName} - COVID Risk Map & Key Metrics`;
```

Comment:
Is short name the correct one to use? Feels like we should use fullName + the title will change a bit

Reply:
ie.-

 Kimball County, Nebraska (NE) - COVID Risk Map & Key Metrics

Nebraska (NE) - COVID Risk Map & Key Metrics

Reply:
right, not quite the same - I guess I would handle it type by type with a good fallback, good catch!